### PR TITLE
MGMT-8784: bump versions - skip x.y.z

### PR DESCRIPTION
**Pull request by danielerez**
Avoid updating x.y.z versions as only x.y should updated to latest.

### CODE REVIEW

File: tools/bump_ocp_releases.py [hunk 1]
```diff
@@ -62,6 +62,7 @@
 OCP_REPLACE_CONTEXT = ['"{version}"', "ocp-release:{version}"]
 
 SKIPPED_MAJOR_RELEASE = ["4.6"]
+MAJOR_MINOR_VERSION_REGEX = re.compile("^([1-9]\d*|0)(\.(([1-9]\d*)|0))$")
```
Comment (osherdp) [inline tools/bump_ocp_releases.py:65]: lint probably wants
```suggestion
MAJOR_MINOR_VERSION_REGEX = re.compile(r"^([1-9]\d*|0)(\.(([1-9]\d*)|0))$")
```

Comment (osherdp) [reply tools/bump_ocp_releases.py:65]: btw, doesn't it work too?
```suggestion
MAJOR_MINOR_VERSION_REGEX = re.compile("^[1-9][0-9]*\.[1-9][0-9]*$")
```

**Review by osherdp — CHANGES_REQUESTED** [review]

File: tools/bump_ocp_releases.py (hunk 1 above)
Comment (danielerez) [reply tools/bump_ocp_releases.py]: > lint probably wants

probably, changed.

Comment (danielerez) [reply tools/bump_ocp_releases.py]: > btw, doesn't it work too?

it won't match 5.0 for example

Comment (osherdp) [reply tools/bump_ocp_releases.py]: right

**Review by osherdp — APPROVED** [review]
/hold
unhold when the job passes

**Pull request merged**